README.md

```diff
@@ -136,6 +136,21 @@ router.get(`/api/tweet/render`, async function(req, res) {
 });
 ```
 
+### Using it on Docker
+You have to follow this instructions (https://github.com/puppeteer/puppeteer/blob/master/docs/troubleshooting.md#running-puppeteer-in-docker) for Docker (Adde to the end of your Dockerfile, there is specific instructiosn for alpine image also) and send this in puppeteerArgs when you call nodeHtmlToImage:
```

Comment:
I think the link is enough, we don't need to tell the user he needs to add it at the end of his Dockerfile 😄 

```suggestion
You have to follow this instructions (https://github.com/puppeteer/puppeteer/blob/master/docs/troubleshooting.md#running-puppeteer-in-docker) to use `node-html-to-image` in  a Docker container. Then, and send this in puppeteerArgs when you call nodeHtmlToImage:
```